core/cloudflow-operator/src/main/scala/cloudflow/operator/Main.scala

```diff
@@ -54,6 +54,7 @@ object Main extends {
 
       Operator.handleAppEvents(client)
       Operator.handleConfigurationUpdates(client)
+      Operator.handleConfigurationInput(client)
```

Comment:
first input secrets are watched, config processed, output secret created, which is watched in `handleConfigurationUpdates`.